server/main.lua

```diff
@@ -5,6 +5,108 @@ local State = {
     IMPOUNDED = 2
 }
 
+---Returns true if the given plate exists
+---@param plate string
+---@return boolean
+local function doesEntityPlateExist(plate)
+    local result = MySQL.scalar.await('SELECT 1 FROM player_vehicles WHERE plate = ?', {plate})
```

Comment:
```suggestion
    local result = MySQL.scalar.await('SELECT 1 FROM player_vehicles WHERE plate = ? LIMIT 1', {plate})
```
Not really needed but the oxmysql docs say to do it so better to maintain the practice

Reply:
Done. Although the docs use it as an example, they don't explicitly recommend it.